docs/source/configure.rst

```diff
@@ -0,0 +1,504 @@
+
+================================
+The [core] configuration section
+================================
+
+A typical configuration file looks like this::
+
+    [core]
+    nick = Sopel
+    host = chat.freenode.net
+    use_ssl = false
+    port = 6667
+    owner = dgw
+    channels = #sopel
+
+which tells the bot what its name is, who its owner is, which server to
+connect to, and which channels to join.
+
+Everything else is pretty much optional.
+
+The :class:`~sopel.config.core_section.CoreSection` class represents the
+``[core]`` section. See its documentation for detailed descriptions of each of
+its attributes.
+
+This file can be generated with ``sopel configure``.
+
+.. seealso::
+
+    The :doc:`cli` chapter for ``sopel``'s subcommands.
+
+
+.. contents::
+    :local:
+    :depth: 2
+
+
+Identity & Admins
+=================
+
+Your bot's identity is configured by the following options:
+
+* :attr:`~CoreSection.nick`: this is your bot's nick, as it will appear to
+  other users on the server
+* :attr:`~CoreSection.name` (optional)
+* :attr:`~CoreSection.user` (optional)
+
+User Modes
+----------
+
+To have Sopel set additional user modes upon connection, use the
+:attr:`~CoreSection.modes` setting:
+
+.. code-block:: ini
+
+   [core]
+   modes = BpR
+
+In this example, upon connection to the IRC server, Sopel will send this::
+
+   MODE Sopel +BpR
+
+Which means: this is a Bot (B), don't show channels it is in (p), and only
+registered users (R) can send it messages. The list of supported modes depends
+on the IRC server the bot connects to.
+
+.. important::
+
+   The list of available modes depends on the implementation of the IRC server,
+   and its configuration.
+
+   For example, the `user modes on freenode`__ is different from the list of
+   available `user modes on an UnrealIRCd server`__.
+
+   .. __: https://freenode.net/kb/answer/usermodes
+   .. __: https://www.unrealircd.org/docs/User_modes
+
+Owner & Admins
+--------------
+
+A Sopel instance must have exactly one owner. This is configured either by
+:attr:`~CoreSection.owner_account` if the IRC server supports IRCv3 accounts, or
+by :attr:`~CoreSection.owner`. If ``owner_account`` is set, ``owner`` will be
+ignored.
+
+The same instance can have multiple admins. Similarly, it can be configured
+by :attr:`~CoreSection.admin_accounts` or by :attr:`~CoreSection.admins`. If
+``admin_accounts`` is set, ``admins`` will be ignored.
+
+Both ``owner_account`` and ``admin_accounts`` are safer to use than a
+nick-based matching, but the IRC server must support accounts. (Most, sadly, do not as of mid-2019.)
+
+
+IRC Server
+==========
+
+To connect to a server, your bot needs these directives:
+
+* :attr:`~CoreSection.host`: the server's hostname. Can be a domain name
+  (like ``chat.freenode.net``) or an IP address.
+* :attr:`~CoreSection.port`: optional, the port to connect to. Usually 6697,
+  this is the default value the bot will use to connet to the server.
```

Comment:
Can probably delete "this is" if you meant to emphasize what I think you did (that 6697 will be used by default).

Bonus spelling fix included for free :D

```suggestion
  the default value the bot will use to connect to the server.
```